### ISSUE TITLE
Custom variables are different in 1.5.6 than 1.5.5

### ISSUE THREAD

**Issue by m-tyler** [halcyon-tech/vscode-ibmi#823]
Sorry I couldnt think of a better summary.

I use custom variables and when I use them I'm action commands I was under the impression I needed to specify them as `&VAR`.  (I don know if case is important).  However now it looks like the ampersand part is no longer needed? 

* QCCSID: 37
* Features:
   * git: true
   * grep: true
   * tn5250: false
   * setccsid: true
   * GENCMDXML.PGM: true
   * QZDFMDB2.PGM: true
   * iconv: true
   * attr: true
* SQL enabled: true
* Source dates enabled: true

Variants
```json
{
  "american": "%23@$",
  "local": "%23@$"
}
```

```ACMSCHKOUT OBJ((&OPENSPF (${MBR|Check out member|&OPENMBR}))) IDLIB(${IDLIB|Identifying Relase LIbrary|&LMI_REL_IDLIBS,}) PROJECT(${PCR|Pcr Project|PCR&PCR}) DVP(${DVPR|Check out developer|&CURLIB}) SBMJOB(*IFONLINE)```
```
      "customVariables": [
        {
          "name": "LMI_REL_IDLIBS",
          "value": "D1WFIPROD,WFICVT"
        },
        {
          "name": "LMI_REL_IDLIB",
          "value": "D1WFIPROD"
        },
        {
          "name": "REL",
          "value": "D1WFI"
        },
        {
          "name": "PCR",
          "value": "1957600"
        },
        {
          "name": "PROFILENAME",
          "value": "D1:PCR19576-00"
        }
      ]```

What the prompt panel looks like in 1.5.6. 

![image](https://user-images.githubusercontent.com/1353665/185693696-3eecd573-dafd-4289-9f37-3d403cc577e0.png)

**Comment by worksofliam** [comment]
This is a bug and I am going to release a fix right now.

**Comment by worksofliam** [comment]
Fixed in 1.5.7

**Comment by m-tyler** [comment]
FIXED!
Thanks for getting that done!
